src/views/ModelPlan/ReadOnly/ModelBasics/index.scss

```diff
@@ -0,0 +1,11 @@
+.read-only-model-plan__timeline__list-item {
+  border-left-width: 0.3rem !important;
+
+  &::before {
+    content: '' !important;
+    width: 1.1rem !important;
+    height: 1.1rem !important;
+    left: 10px !important;
+    margin-top: 0 !important;
+  }
+}
```

Comment:
Had to forcefully remove content (the numbers) and adjust its size and alignment to change  the Process List bullets to match figma

|Stock|Modified|
|---|---|
|![image](https://user-images.githubusercontent.com/2349702/190243092-7657c1cc-972a-4c15-8791-10b6aa49733b.png)|![image](https://user-images.githubusercontent.com/2349702/190243223-703a767e-726d-4436-af11-accdfca925e0.png)|